### PR TITLE
[data-export] Add 'Hide Object Columns' checkbox

### PR DESCRIPTION
**Pull request by guillaumeSF**
## Issue ticket number and link
#352

## Checklist before requesting a review
- [x] I have read and understand the [Contributions section](https://github.com/tprouvot/Salesforce-Inspector-reloaded#contributions)
- [x] Target branch is releaseCandidate and not master
- [x] I have performed a self-review of my code
- [x] I ran the [unit tests](https://github.com/tprouvot/Salesforce-Inspector-reloaded#unit-tests) and my PR does not break any tests
- [x] I documented the changes I've made on the [CHANGES.md](https://github.com/tprouvot/Salesforce-Inspector-reloaded/blob/master/CHANGES.md) and followed actual conventions
- [x] I added a new section on [how-to.md](https://github.com/tprouvot/Salesforce-Inspector-reloaded/blob/master/docs/how-to.md) (optional)

### CODE REVIEW

File: addon/data-export.js [hunk 1]
```diff
@@ -80,6 +80,7 @@ class Model {
     this.winInnerHeight = 0;
     this.queryAll = false;
     this.queryTooling = false;
+    this.prefHideRelations = localStorage.getItem("defaultValueHideObjectNameColumns") == "true"; // default to false
```
Comment (tprouvot) [inline addon/data-export.js:83]: I think you can remove the prefix "defaultValue" and add a suffix to indicate where this option is used like `hideObjectNameColumnsDataExport`

**Comment by tprouvot** [comment]
Hi @guillaumeSF ,
Did you had a chance to see my previous comment ?
We also talked about the how-to section, did you planned to do it or maybe you want me to do it ?

**Comment by guillaumeSF** [comment]
Done @tprouvot !

**Comment by tprouvot** [comment]
Perfect thank yoiu @guillaumeSF !

**Pull request merged**